parsing/pprintast.ml

```diff
@@ -136,6 +136,48 @@ module Doc = struct
 
   let tyvar ppf s =
     Format_doc.fprintf ppf "%s" (tyvar_of_name s)
+
+  (* Expressions are considered nominal if they can be used as the subject of a
+     sentence or action. In practice, we consider that an expression is nominal
+     if they satisfy one of:
+     - Similar to an identifier: words separated by '.' or '#'.
+     - Do not contain spaces when printed.
+     - Is a constant that is short enough.
+  *)
+  let nominal_exp_doc lid t =
```

Comment:
I have the impression that we don't need to delay the dependency on `longident` anymore?
Also since the function is in the `Doc` submodule, the `_doc` suffix seems redundant.

Reply:
Done!